src/orders/utils/data.ts

```diff
@@ -356,17 +360,49 @@ export const isLineAvailableInWarehouse = (
   return false;
 };
 
-export const transformFuflillmentLinesToStockInputFormsetData = (
+export const getLineAvailableQuantityInWarehouse = (
+  line: OrderFulfillLineFragment,
+  warehouse: WarehouseFragment,
+) => {
```

Comment:
```suggestion
): number => {
```

Reply:
There is no need to add typing as the type is inferred automatically.